### DISCUSSION TITLE
[Bug]: Instalação Unit uRESTDWComponentBase was compiled with a different version of uRESTDWConsts.TRESTDWAboutInfo

### DISCUSSION THREAD

**Discussion by paulohostert** [OpenSourceCommunityBrasil/REST-DataWare#199] [Q&A]
### Descrição do bug

Após atualização dos fontes a instalação passou a apresentar o erro com Delphi 11.2:
[dcc32 Fatal Error] uRESTDWPhysicBase.pas(30): F2051 Unit uRESTDWComponentBase was compiled with a different version of uRESTDWConsts.TRESTDWAboutInfo

### Passos para reproduzir o problema

Build do pacotes.
![image](https://user-images.githubusercontent.com/94071196/216838342-8c6a4158-10b2-46b7-a466-2d52fa354868.png)

### Imagens do problema

![image](https://user-images.githubusercontent.com/94071196/216838292-7f1be470-3ca3-44b6-bba5-b50218f717e3.png)

### Versão RDW:

2.1.0 (branch)

### IDE:

Delphi novo

### Sistema Operacional:

Windows

**Comment by mobius1qwe** [comment]
https://github.com/OpenSourceCommunityBrasil/REST-DataWare/wiki/Atualiza%C3%A7%C3%A3o-2.1
Precisa desinstalar o pacote anterior, recompilar e instalar o pacote novo.

**Reply by paulohostert** [reply]
Morbius, desinstalei, e ai tentei instalar novamente o erro persiste.

também passou a ocorrer este erro:
![image](https://user-images.githubusercontent.com/94071196/216859614-78a5887d-94f1-493d-9b18-17d3853b0f61.png)

**Reply by paulohostert** [reply]
Baixa todo o fonte novamente e deu certo!
